resume.html

```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<html>
+    <head>
+            <meta charset="UTF-8">
+            <title>Tina Wilson - Software Developer - Resume</title>
+            <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Oswald:400,700" rel="stylesheet">
+            <link rel="stylesheet" href="style.css">
+            <meta name="viewport" content="initial-scale=1.0,width=device-width" />
+            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+            <link href="https://fonts.googleapis.com/css?family=Merienda|Modern+Antiqua|Roboto+Condensed" rel="stylesheet">
+    </head>
+<body class="resumebody">
+<header><h1>Resume</h1></header>
+<div class="fullmenu">
+        <nav class="menu">
+           <div class="menu_item"><a href="index.html">Home</a></div>
+           <div class="menu_item"><a href="projects.html">Portfolio</a></div>
+           <div class="menu_item"><a href="resume.html">Resume</a></div>
+           <div class="social_item hidden"><a href="https://www.linkedin.com/in/tina-wilson-b973b437/"><a href="#" class="fa fa-linkedin"></a></a></div>
+           <div class="social_item hidden"><a href="https://github.com/twilson16"><i class="fa fa-github"></i></a></div>
+           <div class="social_item hidden"><a class="social" href="mailto:christinawilson6@gmail.com"><a href="#" class="fa fa-google"></a></a></div>
+        </nav>
+   </div>
+<div class="resume">
+    <div class="proskills">
+    <div class="proskills-header"><h2>Professional Skills</h2></div>
+   <div class="proskills-list">
+       <li class="skills-item">Strong communication, organzational, interpersonal, and customer service skills</li>
+       <li class="skills-item">Team Leader for 3.5 years</li>
+       <li class="skills-item">Ability to adapt to changing environments and manage tasks simultaneously</li>
+       <li class="skills-item">Developing and maintaining professional relationships in a variety of settings</li>
+       <li class="skills-item">Problem solving and developing creative solutions</li>
+       <li class="skills-item">Working well independently and as a member of a team</li>
+       <li class="skills-item">Fast Learner</li>
+   </div>
+   </div>
+   </div>
+   <div class="licensure">
+       <p class="licensure-header">Licensure</p>
+       <div class="licensescont">
+        <div class="licenses">Licensed Clinical Social Worker (LCSW)</div><div class="licenses">Licensed Clinical Addiction Specialist (LCAS)</div>
+    </div>
+    </div>
+    <div class="workexperience-header"><h2>Work Experience</h2></div>
+   <div class="workexperience">
+       <div class="eap">
+           <div class="mobile"> 
+           <span class="job"><h3>Workplace Options, LLC</h3></span>
+           <span class="location">Raleigh, NC</span>
+           <div class"jobtitle">Employee Assistance Program Counselor</div>
```

Comment:
You are missing an equals sign here.

Reply:
Good catch! Definitely something I will keep an eye out for in the future! Should I go ahead and make all these changes? I definitely am happy to, just uncertain of the process of the feedback.